package.json

```diff
@@ -24,11 +25,13 @@
   "dependencies": {
     "axios": "^0.19.2",
     "discord.js": "^12.1.1",
+    "dotenv": "^8.2.0",
     "firebase-admin": "^8.12.1",
     "moment-timezone": "^0.5.31",
     "node-schedule": "^1.3.2",
     "pino": "^6.3.2",
-    "pino-pretty": "^4.0.0"
+    "pino-pretty": "^4.0.0",
+    "ts-node-dev": "^1.0.0-pre.62"
```

Comment:
I think `ts-node-dev` should be a dev dependency?

Reply:
@eddiejaoude `devDependencies` is now actually deprecated in npm

Reply:
really?! I don't believe you

Reply:
@eddiejaoude omg, this must have been another package library… yes this can be fixed